api/types/transfer/importexport.proto

```diff
@@ -38,4 +40,15 @@ message ImageExportStream {
 	string stream = 1;
 
 	string media_type = 2;
+
+	// The image names
+	repeated string images = 3;
+	// The specified platforms
+	repeated types.Platform platforms = 4;
+	// Whether to include all platforms
+	bool all_platforms = 5;
+	// Skips the creation of the Docker compatible manifest.json file
+	bool skip_docker_manifest = 6;
```

Comment:
Is this only required for compatibility with older versions of Docker now?

Reply:
Still no version of docker that supports OCI layout.